source/dotnet/Library/AdaptiveCards/AdaptiveHeight.cs

```diff
@@ -51,6 +51,38 @@ public AdaptiveHeight()
         {
         }
 
+        /// <summary>
+        /// Initializes an AdaptiveHeight instance with the given string "auto" |"stretch" or "100px"
+        /// </summary>
+        /// <param name="value">enumeration value or pixel .</param>
+        public AdaptiveHeight(string value)
+        {
+            if (value == null)
```

Comment:
```suggestion
            if (String.IsNullOrEmpty(value))
```

same for `AdaptiveWidth`

Reply:
done